content/4_Gold/LIS.mdx

```diff
@@ -353,6 +362,8 @@ class PCB{
 
 ## Extension - LCS on Permutations
 
+<FocusProblem problem={problems.lisperm} />
+
 Given two permutations of length $N$ ($1\le N \le 10^5$), find the longest common subsequence.
 
 ### Example
```

Comment:
actually, could you remove the sample input explanation here and add it to the statement https://codeforces.com/gym/102951/problem/C?